README.md

```diff
@@ -4,13 +4,44 @@ This tap provides Homebrew formulas for the greptime and gtctl utilities.
 
 ## Installation
 
-```console
+```bash
 brew tap greptimeteam/greptime
 ```
 
 Then, you can install `greptime` or `gtctl`:
 
-```console
+```bash
 brew install greptime
 brew install gtctl
 ```
+
+## Upgrading
+
+To upgrade existing installations to the latest version:
+
+```bash
+brew upgrade greptime
+brew upgrade gtctl
+```
+
+## 🚀 Starting GreptimeDB
+
+Run GreptimeDB in standalone mode with:
+
+```bash
+greptime standalone start \
+  --data-home ./greptimedb_data \
```

Comment:
This is default value, we can remove the --data-home option

Reply:
done